src/main/java/app/Routes.java

```diff
@@ -11,23 +13,23 @@ private Routes() {
     static {
         Router router = new Router();
 
-        router.addMethodToRoute("/simple_get", "GET", Methods.SIMPLE_GET);
-        router.addMethodToRoute("/simple_get", "OPTIONS", Methods.SIMPLE_OPTIONS);
+        router.addMethodToRoute("/simple_get", HTTPMethods.GET, Methods.SIMPLE_GET);
```

Comment:
Definitely! I'm making these changes but keeping the method parameter since I want the user to be able to assign the handler method of their choice to any given route (this is kind of rails-ey I suppose).

Reply:
Ah, I see. I was confused by the wording. I think you're referring to the controller abstraction??